example/src/App.js

```diff
@@ -1,78 +1,132 @@
-import React from "react";
-import { HashRouter as Router, Route, NavLink as Link, Redirect } from "react-router-dom";
+import 'font-awesome/css/font-awesome.css';
+import React from 'react';
+import {
+  HashRouter as Router,
+  Route,
+  NavLink as Link,
+  Redirect,
+} from 'react-router-dom';
+import { Nav } from './components/Nav';
 
 // Routes
-import DyanmicListExample from "./routes/examples/DynamicList";
-import FixedSizeGridApi from "./routes/api/FixedSizeGrid";
-import FixedSizeListApi from "./routes/api/FixedSizeList";
-import FixedSizeGridExample from "./routes/examples/FixedSizeGrid";
-import FixedSizeListExample from "./routes/examples/FixedSizeList";
-import FixedSizeListWithScrollingIndicatorExample from "./routes/examples/FixedSizeListWithScrollingIndicator";
-import ScrollToItemExample from "./routes/examples/ScrollToItem";
+import DyanmicListExample from './routes/examples/DynamicList';
+import FixedSizeGridApi from './routes/api/FixedSizeGrid';
+import FixedSizeListApi from './routes/api/FixedSizeList';
+import FixedSizeGridExample from './routes/examples/FixedSizeGrid';
+import FixedSizeListExample from './routes/examples/FixedSizeList';
+import FixedSizeListWithScrollingIndicatorExample from './routes/examples/FixedSizeListWithScrollingIndicator';
+import ScrollToItemExample from './routes/examples/ScrollToItem';
```

Comment:
All of these formatting changes probably weren't intentional?

Reply:
no they were not intentional.  I'm using vscode and I thought it would have picked up the .prettierrc.  Where is your setting for double quote strings?

Reply:
Hm. I think it's actually a problem with my Code settings. 😅 My only concern at the moment is that it will complicate merging. So I should get it sorted out in master I guess.